### DISCUSSION TITLE
Failed to retrieve contributions. This is likely a GitHub API issue.

### DISCUSSION THREAD

**Discussion by TheToriqul** [DenverCoder1/github-readme-streak-stats#732] [General]
![image](https://github.com/user-attachments/assets/f8582764-f4d2-4e0e-b445-0415fe70cb97)
I am fetching this issue for few days now

**Comment by moved-to-sloviso03** [comment]
I'm having the same issue, did you solve it?

**Reply by TheToriqul** [reply]
Yes, it works sometimes—occasionally it functions properly, but other times it doesn't.

**Reply by muhammad-fiaz** [reply]
> Yes, it works sometimes—occasionally it functions properly, but other times it doesn't.

I believe this issue is caused by the GitHub API rate limit. However, using a GitHub secret token should resolve it if the application supports retrieving the token from the user. Otherwise, the rate limit may occasionally fail.
